e2e/crd_validation_test.go

```diff
@@ -240,6 +240,57 @@ func TestCRDDefaulting(t *testing.T) {
 					},
 				},
 			},
+			"TargetLabels/from-kubernetes-labels": {
+				obj: &monitoringv1.ClusterPodMonitoring{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "nondefault",
+						Namespace: "default",
+					},
+					Spec: monitoringv1.ClusterPodMonitoringSpec{
+						Endpoints: []monitoringv1.ScrapeEndpoint{
+							{
+								Interval: "1m",
+								Port:     intstr.FromString("metrics"),
+							},
+						},
+						TargetLabels: monitoringv1.ClusterTargetLabels{
+							FromPod: []monitoringv1.LabelMapping{
+								{
+									From: "app.kubernetes.io/name",
+									To:   "name",
+								},
+								{
+									From: "app.kubernetes.io/instance",
+									To:   "instance",
```

Comment:
We have to pick any other target name otherwise we have:

```
 crd_validation_test.go:113: Unexpected error: ClusterPodMonitoring.monitoring.googleapis.com "nondefault" is invalid: spec.targetLabels.fromPod[1].to: Invalid value: "string": cannot relabel onto protected label "instance"
 ```